doc/man3/CMS_add1_signer.pod

```diff
@@ -82,6 +82,12 @@ bit AES, 128 bit AES, triple DES, 128 bit RC2, 64 bit RC2, DES and 40 bit RC2.
 If any of these algorithms is not available then it will not be included: for example the GOST algorithms will not be included if the GOST ENGINE is
 not loaded.
 
+Note that in the case of no-signedAttributes that for some hash-less signing
+schemes the given hash B<md> will be set to a hash required by its specification,
+independent of what is provided. This is the case for ML-DSA and EdDSA (RFC 8419).
+For SLH-DSA (RFC 9814), the suggested hash will only be used if none is
+provided.
```

Comment:
This is confusing. I see double `that` on the first line and there are other confusing formulations. Perhaps something like this?
```suggestion
Note that, in the case signedAttributes are not used, for some hash-less signing
schemes the given hash B<md> will be ignored and a hash required by the signing
scheme will be used. This is the case for ML-DSA (RFC 9882) and EdDSA (RFC 8419).
For SLH-DSA (RFC 9814), the scheme-suggested hash will only be used if B<md> is
NULL.
```

However the above text does not seem to match the implementation as it looks like only the EdDSA has `noattr_md_a_must` set.

Reply:
Thanks. I corrected the text to reflect that only EdDSA has a required hash, the other ones have suggested hashes.